client/vitest.config.manual.ts

```diff
@@ -0,0 +1,22 @@
+/// <reference types="vitest/config" />
+
+import path from "path";
+import { defineConfig } from "vite";
+
+export default defineConfig({
+  resolve: {
+    alias: {
+      "@": path.resolve(__dirname, "./src"),
+      "@styles": path.resolve(__dirname, "./src/styles"),
+      "@components": path.resolve(__dirname, "./src/components"),
+    },
+  },
+  test: {
+    exclude: ["node_modules/**"],
+    silent: "passed-only",
+    typecheck: {
+      enabled: true,
+    },
+    printConsoleTrace: true,
+  },
+});
```

Comment:
Spreading our testing config across many files does not scale well. We would have to update config in every file.

This is a use case for [vitest projects](https://vitest.dev/guide/projects)

The main config would be roughly structured something like this:

```typescript
test: {
  exclude: ['node_modules'], // global test settings
  resolve: {},
  projects: [
    {
      extends: true,
      test: {
        name: 'translations',
        include: [],
      }
    },
    {
      extends: true,
      test: {
        name: 'unit',
        include: [],
        exclude: [],
      }
  ]
}
```

We can run test projects like this:
`npm run test --project translations`
`npm run test --project unit`